packages/material-ui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -87,27 +86,37 @@ const PickersPopper: React.FC<PickerPopperProps & WithStyles<typeof styles>> = (
 
     if (open) {
       lastFocusedElementRef.current = document.activeElement;
+
+      // Focus the paper element when popper opens, otherwise onBlur will not be invoked.
```

Comment:
Which `onBlur`?

Reply:
The onBlur on the Paper Component. Or the function handleBlur below. handleBlur will only be called when the Paper component has a focus, i.e. click on the paper or focus it programmatically.